help-beta/astro.config.mjs

```diff
@@ -1,9 +1,41 @@
+import * as fs from "node:fs";
+
 import starlight from "@astrojs/starlight";
 import {defineConfig} from "astro/config";
+import Icons from "unplugin-icons/vite";
 
 // https://astro.build/config
 export default defineConfig({
     base: "help-beta",
+    vite: {
+        plugins: [
+            // eslint-disable-next-line new-cap
+            Icons({
+                compiler: "astro",
+                // Icons had a default scale of 1.2, which was making
+                // them look larger than the text around them.
+                scale: 1,
+                defaultStyle: "display: inline; vertical-align: text-bottom;",
+                customCollections: {
+                    // unplugin-icons has a FileSystemIconLoader which is more
+                    // versatile. But it only supports one directory path for
+                    // a single set of icons. We should start using that loader
+                    // if they add support for multiple paths in the future.
```

Comment:
We can use `FileSystemIconLoader` twice, no?

Reply:
We need both the paths to have the `zulip-icon` naming since that is the naming we follow on our web-app. So we cannot have FileSystemIconLoader twice with different icon naming schemes. Let me know if using it twice meant something else, thank you!